strutil/strutil.go

```diff
@@ -7,6 +7,7 @@ import (
 	"encoding/base64"
 	"encoding/json"
 	"fmt"
+	"regexp"
```

Comment:
There are no tests for the added functions in this file

Reply:
I added test cases for both functions in the same style as the other test were written. I took your example from https://play.golang.com/p/3R-3kk-BO9I as a base to generate the tests.